docs/user-guide/recipes.rst

```diff
@@ -104,6 +104,75 @@ a day.
   Schedule successfully created
 
 
+Uploading Content
+=================
+
+It is easy to upload RPMs, SRPMs and DRPMs from your machine via ``pulp-admin rpm repo uploads rpm`` to pulp.
```

Comment:
After reading this sentence I think I can upload RPMs, SRPMs and DRPMs with the one command.
Maybe drop the last part of the command like this ``pulp-admin rpm repo uploads`` and then say that example below is for RPMs? Or any other way to be clear that there are different commands for each unit type.